src/Data/List/Membership/Propositional/Properties.agda

```diff
@@ -123,10 +126,13 @@ module _ {v : A} where
   ∈-++⁻ : ∀ xs {ys} → v ∈ xs ++ ys → (v ∈ xs) ⊎ (v ∈ ys)
   ∈-++⁻ = Membership.∈-++⁻ (≡.setoid A)
 
+  ∈-++ : v ∈ xs ++ ys ⇔ (v ∈ xs ⊎ v ∈ ys)
```

Comment:
This should probably mention `⇔` in the name (cf. `concat-∈↔`  below)

Reply:
Wouldn't it be better to have a consistent naming convention for bundling introductions & eliminations?
Since intros are named `<property_name>⁺` (modulo left/right qualifiers) and eliminators `<property_name>⁻`, my suggestion was to just drop the polarities (i.e. `<property_name>`) to communicate that this includes both intro & elim. 

Another naming convention that I find intuitive is to explicitly include both polarities: `<property_name>⁺⁻`, e.g. `∈-++⁺⁻`.

The things I don't like about the current naming of such bundles are the following:
- The exact arrow being used will change in each case (whether it's an equivalence  `⇔`, an inverse `↔`, a bi-inverse, etc...); isn't it preferable to have a universal convention that hides such differences (much like we do for intros/elims which might take different forms in each individual case but are always named with `⁺` and `⁻`.
- names like `concat-∈↔` are not immediately linked to the intros/elims that it contains  (i.e. the `<property_name>` part is not retained)

So, if I follow the current practices, I would have to name this `++-∈⇔` :(
I can definitely do that, but I find it ugly and unintuitive.